tcms/testplans/templates/testplans/mutable.html

```diff
@@ -92,7 +112,27 @@
                 </div>
             </div>
 
+            {% if test_plan %}
+                <div class="form-group">
+                    <label class="col-md-2 col-lg-2">{% trans "Notification Include" %}</label>
+                    <div class="col-md-10 col-lg-10">
+                        <div>{{ form.auto_to_plan_author }} {{ form.auto_to_plan_author.label }} {{ form.auto_to_plan_author.errors }}</div>
+                        <div>{{ form.auto_to_case_owner }} {{ form.auto_to_case_owner.label }} {{ form.auto_to_case_owner.errors }}</div>
+                        <div>{{ form.auto_to_case_default_tester }} {{ form.auto_to_case_default_tester.label }} {{ form.auto_to_case_default_tester.errors }}</div>
+                    </div>
+
+                    <label class="col-md-2 col-lg-2">{% trans "Notification Trigger" %}</label>
+                    <div class="col-md-10 col-lg-10">
+                        <div>{{ form.notify_on_plan_update }} {{ form.notify_on_plan_update.label }} {{ form.notify_on_plan_update.errors }}</div>
+                        <div>{{ form.notify_on_case_update }} {{ form.notify_on_case_update.label }} {{ form.notify_on_case_update.errors }}</div>
+                    </div>
+                </div>
+            {% endif %}
```

Comment:
Both the Active switch and the notification settings must also be available when creating a new TestPlan. This will allow the author to turn on/off notifications as they need without having to Save and then Edit the document.

The legacy templates have it wrong so you will have to fix it first. Because this will touch both forms and existing views it will be best if you can split only the Active and Notifications changes into a separate PR before continuing to work on this one.

Other notes:
* These checkboxes should be bootstrap-switch which we use in other places
* Don't use form.field.label - this is inconsistent with the existing forms, many of these labels aren't marked for translation and the text should really be in the templates. Plus this doesn't give you lots of control over how the label is rendered.
* Same comment about form.field instead of crafting the field in HTML and specifying classes for it.